spec/core_ext/string_spec.rb

```diff
@@ -194,6 +194,10 @@
     it "converts an array" do
       expect("[0,1,2,3]".convert_to_value).to eql [0,1,2,3]
     end
+
+    it "just returns the string if something goes wrong" do
+      expect("[.a,2,3]".convert_to_value).to eql "[.a,2,3]"
```

Comment:
Is the spec change enough to capture the regression that he saw? He had a binary string.